reference-implementation/lib/transform-stream.js

```diff
@@ -318,7 +318,12 @@ class TransformStreamDefaultSink {
 
     if (transformStream._backpressure === true) {
       return transformStream._backpressureChangePromise
-          .then(() => TransformStreamTransform(transformStream, chunk));
+          .then(() => {
+            if (transformStream._errored === true) {
+              return Promise.reject(transformStream._storedError);
```

Comment:
I think this will be better as `throw transformStream._storedError`? They're equivalent, but thinking of how we'll write it in spec language, I think we'll say "transforming by a function that throws ... if ..., and otherwise returns ..."

Reply:
Reviewing https://github.com/whatwg/streams/pull/800 I see there is a lot of pre-existing code that does return Promise.reject() from inside fulfillment/rejection handlers. So maybe not worth fixing just this one-off, but instead doing a greater alignment when or slightly before you do the spec translation.

Reply:
It will be easier to do as one big cleanup in a separate pass, so if you don't mind I will do that. I have filed https://github.com/whatwg/streams/issues/804 to track this.